### ISSUE TITLE
Small angle quaternion function reference needs editing

### ISSUE THREAD

**Issue by chutsu** [KumarRobotics/msckf_vio#9]
**TLDR**: There are 2 versions of "Indirect Kalman Filter for 3D Attitude Estimation", the one you reference may not be the "offical one" therefore the equations you referenced could be misleading to others.

In the [`math_utils.hpp`](https://github.com/KumarRobotics/msckf_vio/blob/master/include/msckf_vio/math_utils.hpp) in particular the [`smallAngleQuaternion()` ](https://github.com/KumarRobotics/msckf_vio/blob/master/include/msckf_vio/math_utils.hpp#L63) documentation it lists Trawny and Roumeliotis' "Indirect Kalman Filter for 3D Attitude Estimation" as a source, however the one you referenced is actually not the official document (I don't think).

There appears 2 different versions of the same document, the one you referenced with the additional "A Tutorial for Quaternion Algebra" as a subtitle, and the more accessible version where this subtitle is not on the front page.

(Your reference)
https://pdfs.semanticscholar.org/7d78/9dd851ccf6100a3045ab9347b35eb86a9106.pdf

(The more official "Indirect Kalman Filter for 3D Attitude Estimation" document straight from MARS lab server)
http://mars.cs.umn.edu/tr/reports/Trawny05b.pdf

The difference between the two version is the small angle quaternion equation is no longer Equation (238) and (239) as documented, but as Equation (217) and (218) (in the "more official" techical report).

**Comment by ke-sun** [comment]
Thanks for the clarification!
